gradle.properties

```diff
@@ -4,8 +4,12 @@ org.gradle.welcome=never
 # version=24.5.6-acme
 # versionappendcommit=true
 
+test.maxParallelForks = 1
```

Comment:
Do we want to merge any of the changes in this file, or are they just for your local env? I suspect we want to leave them unchanged?

Reply:
Yeah, I don't remeber to change gradle.properties for any reason except for my local tests, so they should be unchanged...

Reply:
Reverted to the _main_ version.